libs/chlorophyll/scss/components/wizard/in-page/_mixins.scss

```diff
@@ -123,4 +115,30 @@
     display: flex;
     background-color: tokens.get(tokens.$green, 1);
   }
+
+  &.completed {
+    &.editable {
+      #{$prefix}__header__edit {
+        grid-area: edit;
+        display: none;
+        justify-content: end;
+        @media (min-width: 576px) {
```

Comment:
Hey, thank you for the PR!

Here I think we should use one of the predefined media query mixins, something like:
```scss
@include common.media-breakpoint-up(sm) {
  display: flex;
}
```
Same thing applies a few lines down in this file.

Otherwise looks great!